File: from-a-to-z/__tests__/fromAToZ.test.js
const { gimmieTheLetters } = require('../fromAToZ');

describe('Given a string indicating a range of letters, return a string which includes all the letters in that range, including the last letter', () => {
    test('Given "a-b", return "ab"', () => {
        const string = "a-b";
        expect(gimmieTheLetters(string)).toBe("ab");
    });
    test('Given "A-B", return "AB"', () => {
        const string = "A-B";
        expect(gimmieTheLetters(string)).toBe("AB");
    });
    test('Given "a-z", return "abcdefghijklmnopqrstuvwxyz"', () => {
        const string = "a-z";
        expect(gimmieTheLetters(string)).toBe("abcdefghijklmnopqrstuvwxyz");
    });
    test('Given "h-o", return "hijklmno"', () => {
        const string = "h-o";
        expect(gimmieTheLetters(string)).toBe("hijklmno");
    });
    test('Given "Q-Z", return "QRSTUVWXYZ"', () => {
        const string = "Q-Z";
        expect(gimmieTheLetters(string)).toBe("QRSTUVWXYZ");
    });
    test('Given "J-J", return "J"', () => {
        const string = "J-J";
        expect(gimmieTheLetters(string)).toBe("J");
    });
});


//https://www.codewars.com/kata/6512b3775bf8500baea77663/train/javascript

File: python-venv/from_a_to_z/from_a_to_z_test.py
from from_a_to_z import gimmie_the_letters

def test_1():
    string = "a-b"
    assert gimmie_the_letters(string) == "ab"

def test_2():
    string = "A-B"
    assert gimmie_the_letters(string) == "AB"

def test_3():
    string = "a-z"
    assert gimmie_the_letters(string) == "abcdefghijklmnopqrstuvwxyz"

def test_4():
    string = "h-o"
    assert gimmie_the_letters(string) == "hijklmno"

def test_5():
    string = "Q-Z"
    assert gimmie_the_letters(string) == "QRSTUVWXYZ"

def test_6():
    string = "J-J"
    assert gimmie_the_letters(string) == "J"

#https://www.codewars.com/kata/6512b3775bf8500baea77663/train/python
    


        #     test.assert_equals(gimme_the_letters("g-i"), "ghi")
        # test.assert_equals(gimme_the_letters("H-I"), "HI")
        # test.assert_equals(gimme_the_letters("y-z"), "yz")
        # test.assert_equals(gimme_the_letters("e-k"), "efghijk")
        # test.assert_equals(gimme_the_letters("a-q"), "abcdefghijklmnopq")
        # test.assert_equals(gimme_the_letters("F-O"), "FGHIJKLMNO")    